src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/Design/Serialization/MemberRelationshipService.cs

```diff
@@ -44,7 +44,9 @@ public abstract class MemberRelationshipService
                 // and not the other as the main constructor performs argument validation.
                 if (source.Owner == null)
                 {
+#pragma warning disable CA2208 // Instantiate argument exceptions correctly, argument not applicable
                     throw new ArgumentNullException(nameof(MemberRelationship.Owner));
```

Comment:
```suggestion
                    throw new ArgumentNullException(nameof(source), SR.Format(SR.PropertyCannotBeNull, nameof(source.Configuration));
```
With a string like

PropertyCannotBeNull: "The '{0}' property is required, and cannot be null."